docs/File-Formats.md

```diff
@@ -1599,6 +1600,31 @@ Allowed values for Arm-level copy-number data are `Loss`, `Gain`, and `Unchanged
 
 Please find example file format here: [Meta file example](https://github.com/cBioPortal/cbioportal-frontend/blob/master/end-to-end-test/local/studies/lgg_ucsf_2014_test_generic_assay/meta_armlevel_CNA.txt) and [Data file example](https://github.com/cBioPortal/cbioportal-frontend/blob/master/end-to-end-test/local/studies/lgg_ucsf_2014_test_generic_assay/data_armlevel_CNA.txt)
 
+### Mutational Signature Data
+Mutational Signature data is a predefined subtype of Generic Assay Data. Setting `generic_assay_type: MUTATIONAL_SIGNATURE` 
+in the meta file will make cBioPortal interpret the data as Mutational Signature data.
+
+#### Mutational Signature meta files
+The mutational signature meta files follow the same convention as the [Generic Assay Meta file](#generic-assay-meta-file),
+however there are some key differences:
+- `genetic_assay_type` should be set to `MUTATIONAL_SIGNATURE`
+- `stable_id` values should end with: `{datatype}_{identifier}`, where:
+  - `datatype` is either `contribution`, `pvalue` or `matrix`
```

Comment:
I guess this would be personal preference. The type of data is a `mutational (count) matrix`, neither of the three words would really give a full representation of the data. But we can change it to `counts` if that improves clarity!

Reply:
@MatthijsPon Just had one question regarding the datatype. I see you will have three new datatypes for mutational signatures data: `contribution`, `pvalue` and `matrix`. Is that mean the mutational signatures data will only be shown in the patient view? I am wondering do you want mutational signatures data to utilize some existing general generic assay data features (only `LIMIT_VALUE`, `CATEGORICAL`, and `BINARY` types can utilize general features for now).

Reply:
Ah yes, this is supposed to be `LIMIT_VALUE` for all the filetypes. I'll add it!

Reply:
I've renamed `matrix` to `counts`, added that datatype should be `LIMIT_VALUE` and rebased on current master